README.md

```diff
@@ -42,6 +42,9 @@ cd substrate-chainlink
 ./setup
 ```
 
+> Before running the setup, you need to change the `ETH_URL` in the **substrate-chainlink/chainlink.env** file.  
+> Replace `YOUR_PROJECT_KEY_FROM_INFURA` by your project key from https://infura.io/  
+
```

Comment:
```suggestion
```

We should simply just set `ETH_DISABLED=true` in the env file instead.

Reply:
Oh, I tested with this setting, without success... Let me check again

Reply:
It will still output some error messages, but it should still technically work :sweat_smile: